### DISCUSSION TITLE
Sound Configuration

### DISCUSSION THREAD

**Discussion by Chromaah** [diamond-rip/Eden#380] [Ideas]
Hey! I thought it'd be a cool idea to allow changing sounds that play for example during countdown, game start, etc. Ive found a lot of use cases for this and i know i wouldnt be the only one wanting it.

I am trying to make custom sounds with a resourcepack but i dont want to change the sounds of that specific one that is played, instead of a different one.

I hope you can add this and so it can be completed soon! :>

**Comment by GoodestEnglish** [comment]
Pretty great idea. Will consider once I finished on working something

**Reply by Chromaah** [reply]
any updates? Just waiting for eden to allow sound configuration, then my server will be done.

**Reply by GoodestEnglish** [reply]
It is being worked on :) If you really need it quick then I can publish a unfinished system which pitch cannot be changed, but sounds are still configurable.

**Reply by Chromaah** [reply]
I can wait a few days, if it will take a few weeks then please post the unfinished system, if its done in a few days, then you can take your time.

**Reply by GoodestEnglish** [reply]
It is finished right now. Just missing the pitch part. I will see if I have time to add it. But it shouldn't take too much time so lets see. If not the next update then 99% the next two updates.

**Reply by Chromaah** [reply]
i mean i dont really need the pitch part but maybe if you dont have time just import it like that in the next update and add the pitch later on

**Comment by Chromaah** [comment]
Hey! Thanks for the update. Can you add the firework blast on match start as a configurable sound as well?

**Reply by GoodestEnglish** [reply]
Adding it along in 1.1.23
